dwitter/feed/views.py

```diff
@@ -138,6 +139,23 @@ class NewDweetFeed(AllDweetFeed):
     feed_name = "new"
 
 
+class NewDweetRssFeed(RssFeed, NewDweetFeed):
+    link = "/rss/new"
+    description = "An RSS feed of the latest Dweets."
+
+    def items(self):
+        return self.get_dweet_list().all()
+
+    def item_title(self, item):
+        return 'Dweet ID #' + item.id + ' by ' + item.author.username
```

Comment:
This line errors since `item.id` is an integer. The following will fix it:
`return 'Dweet ID #' + str(item.id) + ' by ' + item.author.username`